test/testcondition.cpp

```diff
@@ -1928,6 +1931,18 @@ class TestCondition : public TestFixture {
               "}");
         ASSERT_EQUALS("", errout.str());
     }
+
+    void checkConditionIsAlwaysFalse() {
+        check("void f()\n"
```

Comment:
I like short testcases.. I would like that you put the brace after `)`. that saves 1 line. then put `{}` on the same line as the `if` that saves 2 more lines.